### ISSUE TITLE
In substr change in perldelta.pod $lvalue should be $$lvalue

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#12115]
Migrated from [rt.perl.org#113000](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=113000) (status was 'resolved')

Searchable as RT113000$

**Comment by p5pRT** [comment]
### From @shlomif

#### Created by @shlomif




pod/perldelta\.pod in perl\-5\.16\.0 reads&#8203;:

&nbsp;    my $string = "string";
&nbsp;    my $lvalue = \\substr $string\, \-4\, 2;
&nbsp;    print $lvalue\, "\\n"; \# prints "ri"
&nbsp;    $string = "bailing twine";
&nbsp;    print $lvalue\, "\\n"; \# prints "wi"; used to print "il"

The "$lvalue" text on the third anf fifth lines should be replaced with
"$$lvalue"\. Thanks to mauke on \#p5p on irc\.perl\.org for clarifying that\.

<details><summary>Perl Info</summary>

```

Flags:
    category=docs
    severity=medium

Site configuration information for perl 5.16.0:

Configured by shlomif at Fri May 11 11:34:51 IDT 2012.

Summary of my perl5 (revision 5 version 16 subversion 0) configuration:
   
  Platform:
    osname=linux, osvers=3.3.4-desktop-1.mga2, archname=x86_64-linux
    uname='linux telaviv1.shlomifish.org 3.3.4-desktop-1.mga2 #1 smp sat apr 28 01:29:50 utc 2012 x86_64 x86_64 x86_64 gnulinux '
    config_args='-de -Dprefix=/home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0'
    hint=recommended, useposix=true, d_sigaction=define
    useithreads=undef, usemultiplicity=undef
    useperlio=define, d_sfio=undef, uselargefiles=define, usesocks=undef
    use64bitint=define, use64bitall=define, uselongdouble=undef
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='cc', ccflags ='-fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64',
    optimize='-O2',
    cppflags='-fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include'
    ccversion='', gccversion='4.6.3', gccosandvers=''
    intsize=4, longsize=8, ptrsize=8, doublesize=8, byteorder=12345678
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=16
    ivtype='long', ivsize=8, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=8, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -fstack-protector -L/usr/local/lib'
    libpth=/usr/local/lib /lib/../lib64 /usr/lib/../lib64 /lib /usr/lib /lib64 /usr/lib64 /usr/local/lib64
    libs=-lnsl -lgdbm -ldb -ldl -lm -lcrypt -lutil -lc -lgdbm_compat
    perllibs=-lnsl -ldl -lm -lcrypt -lutil -lc
    libc=/lib/libc-2.14.1.so, so=so, useshrplib=false, libperl=libperl.a
    gnulibc_version='2.14.1'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-Wl,-E'
    cccdlflags='-fPIC', lddlflags='-shared -O2 -L/usr/local/lib -fstack-protector'

Locally applied patches:
    RC0


@INC for perl 5.16.0:
    /home/shlomif/apps/perl/modules/lib/perl5/site_perl/5.12.2
    /home/shlomif/apps/perl/modules/lib/perl5/site_perl/5.14.2
    /home/shlomif/apps/perl/modules/lib/site_perl/5.12.2
    /home/shlomif/apps/perl/modules/lib/site_perl/5.14.2
    /home/shlomif/apps/perl/modules/lib/perl5/5.12.2
    /home/shlomif/apps/perl/modules/lib/perl5/5.14.2
    /home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0/lib/site_perl/5.16.0/x86_64-linux
    /home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0/lib/site_perl/5.16.0
    /home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0/lib/5.16.0/x86_64-linux
    /home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0/lib/5.16.0
    .


Environment for perl 5.16.0:
    HOME=/home/shlomif
    LANG=en_GB.UTF-8
    LANGUAGE=en_GB:en
    LC_ADDRESS=en_US.UTF-8
    LC_COLLATE=en_US.UTF-8
    LC_CTYPE=en_US.UTF-8
    LC_IDENTIFICATION=en_GB.UTF-8
    LC_MEASUREMENT=en_GB.UTF-8
    LC_MESSAGES=en_US.UTF-8
    LC_MONETARY=en_US.UTF-8
    LC_NAME=en_GB.UTF-8
    LC_NUMERIC=en_GB.UTF-8
    LC_PAPER=en_US.UTF-8
    LC_SOURCED=1
    LC_TELEPHONE=en_US.UTF-8
    LC_TIME=en_GB.UTF-8
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/home/shlomif/apps/perl/perlbrew/bin:/home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0/bin:/home/shlomif/Download/unpack/graphics/fop/fop-0.93:/home/shlomif/apps/perl/modules/local/bin:/home/shlomif/apps/perl/modules/bin:/home/shlomif/apps/latemp/bin:/home/shlomif/apps/test/quadpres/bin:/home/shlomif/bin:/home/shlomif/apps/apache-maven/apache-maven-2.1.0//bin:/usr/local/bin:/bin:/usr/bin:/usr/games:/usr/lib64/qt4/bin
    PERL5LIB=/home/shlomif/apps/perl/modules/lib/perl5/site_perl/5.12.2:/home/shlomif/apps/perl/modules/lib/perl5/site_perl/5.14.2:/home/shlomif/apps/perl/modules/lib/site_perl/5.12.2:/home/shlomif/apps/perl/modules/lib/site_perl/5.14.2:/home/shlomif/apps/perl/modules/lib/perl5/5.12.2:/home/shlomif/apps/perl/modules/lib/perl5/5.14.2
    PERLBREW_MANPATH=/home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0/man
    PERLBREW_PATH=/home/shlomif/apps/perl/perlbrew/bin:/home/shlomif/apps/perl/perlbrew/perls/perl-5.16.0-RC0/bin
    PERLBREW_PERL=perl-5.16.0-RC0
    PERLBREW_ROOT=/home/shlomif/apps/perl/perlbrew
    PERLBREW_VERSION=0.42
    PERL_AUTOINSTALL=--skipdeps --alldeps
    PERL_BADLANG (unset)
    PERL_MM_USE_DEFAULT=1
    SHELL=/bin/bash

```
</details>

**Comment by p5pRT** [comment]
### From @rjbs

Thanks\, fixed in 5bbddb3

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

Thanks\, fixed in 5bbddb3

**Comment by p5pRT** [comment]
@rjbs - Status changed from 'new' to 'resolved'
